Tuturial-Structure-Template.md

```diff
@@ -1,68 +1,178 @@
-# Title #
-[header image]
+>The title describes a task that you want to do. For example: **Implement step-up authentication**.
+> 
+> To write a page, use the style and tone suggest in the style guide.
 
-## Intro ##
-> (do not include section title) 
+# Title
 
-Write a paragraph that introduces the topic. Explain why the content is relevant, and make sure to introduce the concepts you plan to cover.
- 
-## Prerequisites ##
-> (use as section title)
+> Explain why the content is relevant and what the user will achieve.
+> 
+> For example:
 
-List what the reader needs to know, as well as packages or SDKS that must be available, to successfully work through the tutorial. Keep the list short, but be sure to include language and concepts the reader should be familiar with.  
+Step-up authentication provides a higher level of assurance that a person is who they claim to be. When a user tries to perform a task, they are challenged to produce additional forms of authentication. In this case, when a user who is already logged into your Agora client tries to connect to a highly secure conference, the back-end system sends a one-time sms-code to a registered mobile device which the user enters in your Agora Client.
 
-**IMPORTANT**: The last prerequisite will always be a link to the _“How to get started with Agora”_ tutorial using the url:
-```
-https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=
-```
+This page shows you how to add this functionality to you Agora Client and security infrastructure. 
```

Comment:
This document?

Reply:
Hmmmmm, I am not a fan. My preference is for _This section_. However, as this is an "everything in one page" template, we are emphasising that it is all in one page.

Reply:
Page is fine. I was thinking about the opening sentence for the third-level title. "This section uses the Web client as an example to show how to use a token for client-side user authentication."